documentation/user_docs/docs/manual/Reshaping_etc.rst

```diff
@@ -1,196 +1,282 @@
-#############
-Reshaping etc
-#############
+#####################
+Other shape functions
+#####################
 
-replicate
-=========
+``replicate``
+=============
 
-This is a function to create a higher dimensional dataset by replicating along an additional axis. For example a d2d, which is essentially a matrix of intensity, can be replicated to make a d3d (an array of matrices) where the nth matrix is the same as the original one, or a d4d. In Matlab syntax this means e.g.
+This is a function to create a higher dimensional dataset by replicating along an additional axis. For example a d2d,
+which is essentially a matrix of intensity, can be replicated to make a d3d (an array of matrices) where the nth matrix
+is the same as the original one, or a d4d.
 
-::
+In Matlab syntax this means e.g.
 
-   Oldmat=[1 2; 3 4];
-   Newmat(:,:,1) = Oldmat;
-   Newmat(:,:,2) = Oldmat;
+::
 
-and so on.
+   oldmat = [1 2; 3 4];
+   newmat = repmat(oldmat, 1, 1, 2);
```

Comment:
While this is better formal matlab, the original Oldmat/Newmat example is much clearer about what replicate does. If you want to add a note that this is not best Matlab practice, that would be ok.